docs/ops/state_backends.md

```diff
@@ -124,7 +124,7 @@ RocksDBStateBackend is currently the only backend that offers incremental checkp
 ## Configuring a State Backend
 
 State backends can be configured per job. In addition, you can define a default state backend to be used when the
-job does not explicitly define a state backend.
+job does not explicitly define a state backend. Besides, state backend configured per-job will overwrite the default state backend configured in `flink-conf.yaml`
```

Comment:
I don't believe implying something is enough.

My teammates and I went through this piece of documentation several times when trying to enable checkpoints, but couldn't figure out the exact configurations and the relationship among several config/code keys. Until we shout out in user list http://apache-flink-user-mailing-list-archive.2336050.n4.nabble.com/confusing-RocksDBStateBackend-parameters-td13810.html

Thus I believe the explicit explanation is absolutely necessary.